pkg/gitreceive/build.go

```diff
@@ -112,6 +110,20 @@ func build(
 		}
 	}
 
+	_, disableCaching := appConf.Values["DEIS_DISABLE_CACHE"]
```

Comment:
Any other uses of these feature flags? Any pros/cons of using true/false vs. set/unset?